content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -150,7 +150,34 @@ contains a single `from` element allowing connections from Pods with the label `
   ...
 ```
 
-contains two elements in the `from` array, and allows connections from Pods in the local Namespace with the label `role=client`, *or* from any Pod in any namespace with the label `user=alice`.
+contains two elements in the `from` array, and allows connections from Pods in the local Namespace with the label `role=client`, *or* from any Pod in any namespace with the label 
```

Comment:
The previous wording made sense as a sentence, but now does not.

Reply:
Thanks for pointing this out. Addressed in new PR..

Reply:
@sftim I basically wanted to highlight the fact that, there was no example on how a configuration with different ports would look like.